### PR TITLE
Included the implementation builder for summaries

### PR DESCRIPTION
**Pull request by Mapno**
This pull request adds `prometheu.Summary`s to the library.

### CODE REVIEW

File: prometheusvanilla/builders.go [hunk 1]
```diff
@@ -97,3 +123,16 @@ func bucketsFromTag(tag reflect.StructTag) ([]float64, error) {
 func DefaultBuckets() []float64 {
 	return []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25}
 }
+
+func maxAgeFromTag(tag reflect.StructTag) (time.Duration, error) {
+	maxAgeString, ok := tag.Lookup("max_age")
+	if !ok {
+		return 0, nil
+	}
+	var maxAgeDuration time.Duration
```
Comment (colega) [inline prometheusvanilla/builders.go]: Why do you need this?

**Comment by colega** [comment]
Can you update the `README.md` please? 🙏

**Comment by Mapno** [comment]
> Can you update the `README.md` please? 🙏

Done in d2767436fd296f4de5f136e1d6cd39cf2e93e690, thanks for the suggestion.

File: prometheusvanilla/builders.go (hunk 1 above)
Comment (Mapno) [reply prometheusvanilla/builders.go]: I don't. Removed it in d2767436fd296f4de5f136e1d6cd39cf2e93e690, thanks!

File: prometheusvanilla/builders.go [hunk 2]
```diff
@@ -97,3 +123,15 @@ func bucketsFromTag(tag reflect.StructTag) ([]float64, error) {
...
 }
+
+func maxAgeFromTag(tag reflect.StructTag) (time.Duration, error) {
+	maxAgeString, ok := tag.Lookup("max_age")
+	if !ok {
+		return 0, nil
+	}
+	maxAgeDuration, err := time.ParseDuration(maxAgeString)
+	if err != nil {
+		return 0, fmt.Errorf("invalid time duration specified: %s", err)
```
Comment (colega) [inline prometheusvanilla/builders.go]:
```suggestion
		return 0, fmt.Errorf("invalid max_age tag specified: %s", err)
```

**Pull request merged**